Source/WebKit/UIProcess/API/Cocoa/_WKWebExtensionContext.h

```diff
@@ -25,6 +25,8 @@
 
 #import <WebKit/WKWebExtensionContextPrivate.h>
 
+#define HAVE_WK_WEB_EXTENSION_CONTEXT_ARRAY_BASED_DID_SELECT_TABS 1
```

Comment:
Was this change intentional?

Reply:
Oh, I see this is used for staging internally.

Reply:
Yep